scripts/kubecf-build.sh

```diff
@@ -61,6 +61,8 @@ done
 echo "operatorChartUrl: \"${CF_OPERATOR_URL//\{version\}/${CF_OPERATOR_VERSION}}\"" > "${HELM_DIR}/Metadata.yaml"
 
 ruby rules/kubecf/create_sample_values.rb "${HELM_DIR}/values.yaml" "${HELM_DIR}/sample-values.yaml"
+MODE=check ruby rules/kubecf/create_sample_values.rb "${HELM_DIR}/values.yaml" "${HELM_DIR}/sample-values.yaml"
```

Comment:
Should this be part of lint rather than build?

Reply:
Could be.
This check needs to be run after the `sample-values` creation, which needs a build folder though. It could be added as a target of `make lint`, but then we are duplicating the creation and handling of the temporal folders. 

Maybe this is better solved when we remove Bazel and refactor `rules/`.